### PR TITLE
Adding ability to stop stream by stream_id

### PR DESCRIPTION
**Pull request by clone1018**

### CODE REVIEW

File: lib/glimesh/resolvers/streams_resolver.ex [hunk 1]
```diff
@@ -38,13 +38,23 @@ defmodule Glimesh.Resolvers.StreamsResolver do
     {:error, "Access denied"}
   end
 
+  def end_stream(_parent, %{stream_id: stream_id}, %{context: %{is_admin: true}}) do
+    stream = Streams.get_stream!(stream_id)
+
+    Streams.end_stream(stream)
+  end
+
   def end_stream(_parent, %{channel_id: channel_id}, %{context: %{is_admin: true}}) do
```
Comment (haydenmc) [inline lib/glimesh/resolvers/streams_resolver.ex:47]: I still think it's dangerous to allow ending of arbitrary stream instances by channel ID - if we allow this, we ought to plumb down some notification to anyone listening (FTL instances) that the stream IDs they're operating on have been invalidated so they don't just chug along happily not knowing that the service has decided that they're gone.

File: lib/glimesh/resolvers/streams_resolver.ex [hunk 2]
```diff
@@ -38,13 +38,23 @@ defmodule Glimesh.Resolvers.StreamsResolver do
...
     {:ok, stream}
   end
 
+  def end_stream(_parent, _args, %{context: %{is_admin: true}}) do
+    {:error, "Must specify channelId or streamId"}
+  end
+
   def end_stream(_parent, _args, _resolution) do
     {:error, "Access denied"}
   end
```
Comment (haydenmc) [inline lib/glimesh/resolvers/streams_resolver.ex:60]: Can we get this for `log_stream_metadata` as well?

**Review by haydenmc — COMMENTED** [review]
Couple comments, looks good

File: lib/glimesh/resolvers/streams_resolver.ex (hunk 2 above)
Comment (clone1018) [reply lib/glimesh/resolvers/streams_resolver.ex:53]: Added stream_id as the only allowed input.

**Review by haydenmc — APPROVED** [review]

**Pull request merged**